modules/media/jsx/mediaIndex.js

```diff
@@ -56,6 +54,65 @@ class MediaIndex extends React.Component {
     this.refs.mediaFilter.clearFilter();
   }
 
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    const hiddenHeaders = ['Cand ID', 'Session ID', 'Hide File', 'File Type'];
+
+    // If a column if set as hidden, don't display it
+    if (hiddenHeaders.indexOf(column) > -1) {
+      return null;
+    }
+
+    // Create the mapping between rowHeaders and rowData in a row object.
+    let row = {};
+    rowHeaders.forEach(function(header, index) {
+      row[header] = rowData[index];
+    }, this);
```

Comment:
Not sure if we are moving towards an arrow function standard, but it may be useful to start writing functions like this as:
`rowHeaders.forEach((header, index) => row[header] = rowData[index]);`

Reply:
Does it make sense to use an arrow function here? It's setting data, not being used as a lambda function.

I'll change it if the people who do more javascript than I do think it's the right way to go (@intralizee @davidblader @anyoneelsethatimmissing)

Reply:
I changed it since it's already 2-0 in favour of the arrow function..

Reply:
after some consideration, i think conventional function syntax should be the way to go

Reply:
@driusan use of arrow function here is fine - there's just a syntax error. there should be an extra ; before the last closing parenthesis i.e.

`rowHeaders.forEach((header, index) => {
          row[header] = rowData[index];
});`

but the closing semi-colon is missing here after the command

Reply:
@HenriRabalais